### PR TITLE
Minor edits to Data Streams docs 

### PR DESCRIPTION
**Pull request by khadni**

### CODE REVIEW

File: src/content/data-streams/index.mdx [hunk 1]
```diff
@@ -68,13 +68,13 @@ One example of how to use Data Streams is in a decentralized exchange. An exampl
 1. A user initiates a trade by confirming an `initiateTrade` transaction in their wallet.
 1. The onchain contract for the decentralized exchange responds by emitting a Log Trigger event.
 1. The Chainlink Automation upkeep monitors the contract for the event. When Automation detects the event, it runs the `checkLog` function specified in the upkeep contract. The upkeep is defined by the decentralized exchange.
-1. The `checkLog` function emits an event called `StreamsLookup`.
-1. Data Streams detects the `StreamsLookup` event and returns the requested report in the `checkCallback` function for Chainlink Automation.
+1. The `checkLog` function uses a `revert` with a custom error called `StreamsLookup`. This approach, aligning with [EIP-3668](https://eips.ethereum.org/EIPS/eip-3668#use-of-revert-to-convey-call-information), conveys necessary information through the data in the `revert` custom error.
```
Comment (dwightjl) [inline src/content/data-streams/index.mdx:71]:
```suggestion
1. The `checkLog` function uses a `revert` with a custom error called `StreamsLookup`. This approach aligns with [EIP-3668](https://eips.ethereum.org/EIPS/eip-3668#use-of-revert-to-convey-call-information) and conveys the required information through the data in the `revert` custom error.
```

Avoiding gerunds makes sentences easier to read and usually easier to translate.

File: src/content/data-streams/release-notes.mdx [hunk 2]
```diff
@@ -12,8 +12,8 @@ whatsnext:
 
 import { Aside } from "@components"
 
-<Aside type="note" title="Early Access">
-  Data Streams is available on Arbitrum Mainnet and Arbitrum Goerli in Early Access.{" "}
+<Aside type="note" title="MAINNET EARLY ACCESS">
```
Comment (dwightjl) [inline src/content/data-streams/release-notes.mdx:15]:
```suggestion
<Aside type="note" title="Mainnet Early Access">
```

Make this consistent even though callout titles are converted to all caps in the public docs. If the style changes in the future, it's easier to maintain.

File: src/content/data-streams/reference/interfaces.mdx [hunk 3]
```diff
@@ -6,8 +6,8 @@ title: "Interfaces"
 
 import { Aside, CodeSample } from "@components"
 
-<Aside type="note" title="Early Access">
-  Data Streams is available on Arbitrum Mainnet and Arbitrum Goerli in Early Access.{" "}
+<Aside type="note" title="MAINNET EARLY ACCESS">
```
Comment (dwightjl) [inline src/content/data-streams/reference/interfaces.mdx:9]:
```suggestion
<Aside type="note" title="Mainnet Early Access">
```

Make this consistent even though callout titles are converted to all caps in the public docs. If the style changes in the future, it's easier to maintain.

File: src/features/data-streams/common/gettingStarted.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+This guide shows you how to read data from a Data Stream and validate and store the answer onchain. This example uses a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. For this example, the log trigger comes from a simple emitter contract. Chainlink Automation then uses `StreamsLookup` to retrieve a signed report from the Data Streams Engine, return the data in a callback, and run the [`performUpkeep` function](/chainlink-automation/reference/automation-interfaces#performupkeep-function-for-log-triggers) on your registered upkeep contract. The `performUpkeep` function calls the `verify` function on the verifier contract.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:3]:
```suggestion
This guide shows you how to read data from a Data Stream, validate the answer, and store the answer onchain. This example uses a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. For this example, the log trigger comes from a simple emitter contract. Chainlink Automation then uses `StreamsLookup` to retrieve a signed report from the Data Streams Engine, return the data in a callback, and run the [`performUpkeep` function](/chainlink-automation/reference/automation-interfaces#performupkeep-function-for-log-triggers) on your registered upkeep contract. The `performUpkeep` function calls the `verify` function on the verifier contract.
```

File: src/features/data-streams/common/gettingStarted.mdx [hunk 5]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
+     src="/images/data-streams/getting-started/deployed-upkeep-1.webp"
+     alt="Chainlink Data Streams Remix Deployed Upkeep Contract"
+     style="max-width: 70%;"
+   />
 
-1. Select the Arbitrum Sepolia network in MetaMask.
+## Deploy the emitter contract
 
-1. Open the example emitter in Remix.
+This contract emits logs that trigger the upkeep. This code can be part of your dApp. For example, you might emit log triggers when your users initiate a trade or other action requiring data retrieval. For this Getting Started guide, we will use a very simple emitter so you can test the upkeep and data retrieval.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:82]:
```suggestion
This contract emits logs that trigger the upkeep. This code can be part of your dApp. For example, you might emit log triggers when your users initiate a trade or other action requiring data retrieval. For this Getting Started guide, use a very simple emitter so you can test the upkeep and data retrieval.
```

Avoid "we" and just use imperative sentences for simplicity.

File: src/features/data-streams/common/gettingStarted.mdx [hunk 6]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
+
+   <ClickToZoom
+     src="/images/data-streams/getting-started/compile-logemitter.webp"
+     alt="Chainlink Data Streams Remix Compile Log Emitter Contract"
+     style="max-width: 70%;"
+   />
+
+1. Open MetaMask and make sure the network is still set to _Arbitrum Sepolia_.
+
+1. In Remix, under the **Deploy & Run Transactions** tab, ensure the **Environment** is still set to _Injected Provider - MetaMask_.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:98]:
```suggestion
1. On the **Deploy & Run Transactions** tab in Remix, ensure the **Environment** is still set to _Injected Provider - MetaMask_.
```

This one is just a style choice, but I think it simplifies the sentence.

File: src/features/data-streams/common/gettingStarted.mdx [hunk 7]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
+     src="/images/data-streams/getting-started/emitter-emitlog.webp"
+     alt="Chainlink Data Streams Emit Log"
+     style="max-width: 70%;"
+   />
 
-1. Go to [sepolia.arbiscan.io](https://sepolia.arbiscan.io/).
+1. Click the `emitLog` button to call the function and emit a log. MetaMask prompts you to accept the transaction.
 
-1. Under the **Deployed Contracts** list, click the **emitLog** button to run the function and emit a log. MetaMask prompts you to accept the transaction.
+After the transaction is complete, the log is emitted, and the upkeep is triggered. You can find the upkeep transaction hash at [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia). Check to make sure the transaction is successful.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:178]:
```suggestion
After the transaction is complete, the log is emitted, and the upkeep is triggered. You can find the upkeep transaction hash in the [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia). Check to make sure the transaction is successful.
```

File: src/features/data-streams/common/gettingStarted.mdx [hunk 8]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
-After the transaction is complete, the log is emitted and the upkeep is triggered. You can find the upkeep transaction hash at [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia). Check to make sure the transaction is successful.
+    <ClickToZoom src="/images/data-streams/getting-started/request-fulfilled.webp" />
 
 ## View the retrieved price
 
-The retrieved price is stored as a variable in the contract and is also emitted in the logs.
+The retrieved price is stored as a variable in the contract and emitted in the logs.
 
-1. In Remix, go to the **Deploy & Run** tab.
+1. In Remix, under the **Deploy & Run Transactions** tab, expand your upkeep contract in the **Deployed Contracts** section.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:186]:
```suggestion
1. On the **Deploy & Run Transactions** tab in Remix, expand the details of your upkeep contract in the **Deployed Contracts** section.
```

Style preference

File: src/features/data-streams/common/gettingStarted.mdx [hunk 9]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
 
-1. Click the `last_retrieved_price` variable to view the retrieved price.
+   <ClickToZoom
+     src="/images/data-streams/getting-started/deployed-upkeep-2.webp"
+     alt="Chainlink Data Streams Deployed Upkeep"
+     style="max-width: 70%;"
+   />
 
-Alternatively, you can view the price emitted in the logs for your upkeep transaction. You can find the upkeep transaction hash at [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia) and view the logs in the explorer.
+Alternatively, you can view the price emitted in the logs for your upkeep transaction. You can find the upkeep transaction hash at [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia) and view the logs in the [Arbitrum Sepolia explorer](https://sepolia.arbiscan.io/).
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:196]:
```suggestion
Alternatively, you can view the price emitted in the logs for your upkeep transaction. You can find the upkeep transaction hash at [Chainlink Automation UI](https://automation.chain.link/arbitrum-sepolia) and view the transaction logs in the [Arbitrum Sepolia explorer](https://sepolia.arbiscan.io/).
```

File: src/features/data-streams/common/gettingStarted.mdx [hunk 10]
```diff
@@ -14,107 +14,191 @@ Read data from a Data Stream and validate the answer onchain. This example uses
...
+The example code you deployed has all the interfaces and functions required to work with Chainlink Automation as an upkeep contract. It follows a similar flow to the trading flow in the [Architecture](/data-streams#example-trading-flow) documentation but uses a basic log emitter to simulate the client contract that would initiate a `StreamsLookup`. After the contract receives and verifies the report, `performUpkeep` stores the price from the report in the `last_retrieved_price` and emits a `PriceUpdate` log message with the price. You could modify this to use the data in a way that works for your specific use case and application.
 
-The code example uses `revert` with `StreamsLookup` to convey call information about what streams to retrieve. See the [eip-3668 rationale](https://eips.ethereum.org/EIPS/eip-3668#rationale) for more information on the use of `revert` in this way.
+The code example uses `revert` with `StreamsLookup` to convey call information about what streams to retrieve. See the [eip-3668 rationale](https://eips.ethereum.org/EIPS/eip-3668#rationale) for more information on `revert` use in this way.
```
Comment (dwightjl) [inline src/features/data-streams/common/gettingStarted.mdx:202]:
```suggestion
The code example uses `revert` with `StreamsLookup` to convey call information about what streams to retrieve. See the [EIP-3668 rationale](https://eips.ethereum.org/EIPS/eip-3668#rationale) for more information about how to use `revert` in this way.
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Review by dwightjl — APPROVED** [review]

**Comment by Spunsd** [comment]
Hello

**Pull request merged**